### DISCUSSION TITLE
How to change Stick (mouse) sensitivity by holding a key?

### DISCUSSION THREAD

**Discussion by geextahslex** [AntiMicroX/antimicrox#876] [Q&A]
Hi,
I want to have a less sensitive stick (assigned to mouse movement) as default but I would like to speed it up when Im in menus/inventory etc.
So I thought by holding a button (left bumper or so) change the sensitivity to make the experience more fluent.
You could also see this as changing the mouse dpi by holding a button.
I hope this is understandable.

Thanks for any advice :)

**Comment by MrChillings** [comment]
hi geextahslex,

I am not sure (new to antimicrox), but i think you can achieve this with Mouse Mod in the advanced assignment.

I have tried this with mouse mode on spring. but obviously its not working as that is a relative stick to screen situation.

might work under cursor mode. haven't done that yet.

**Comment by LucasAMV** [comment]
1 - In the main window select the controller button you want to map to change the mouse sensitivity
2 - In the new window that allow you to select keyboard or mouse select any key for now, for example 'Z'. This will allow you to use the "Advanced" button on the bottom left, a new window will open
3 - In the new window, click on the white area from the displayed key you selected ('Z'), it will turn blue
4 - In the combo box below with the value "Blank or KB/M" change to "Mouse Mod", an input field will appear below, enter the value you want. A value < 100 will slow down sensitivity, > 100 will speed it up
5 - After inserting the value, click the button "Insert" on the right, check above if the value "Mouse Mod" replaced the key 'Z' from _Step 2_
6 - If the value changed you can click close in this window and in the next one too
7 - Try moving the cursor and while it is moving press the controller button you selected, it should be working.

[Screencast from 2024-09-20 16-25-20.webm](https://github.com/user-attachments/assets/685dbc24-9aa7-4107-8140-2c426263847f)
